water_neuron/src/management.rs

```diff
@@ -311,6 +311,30 @@ pub async fn split_neuron(
     .await
 }
 
+pub async fn stop_dissolvement(neuron_id: NeuronId) -> Result<ManageNeuronResponse, String> {
+    manage_neuron(
+        Command::Configure(Configure {
+            operation: Some(Operation::StopDissolving(StopDissolving {})),
+        }),
+        NeuronNonceOrId::Id(neuron_id),
+    )
+    .await
+}
+
+pub async fn merge_neuron(
+    neuron_nonce: u64,
+    neuron_id: NeuronId,
+) -> Result<ManageNeuronResponse, String> {
+    assert!(neuron_id.id != SIX_MONTHS_NEURON_ID && neuron_id.id != EIGHT_YEARS_NEURON_ID);
+    manage_neuron(
```

Comment:
We are always going to merge into the 6-months, I would rather not take the neuron_nonce as a parameter.